jwst/tweakreg/tweakreg_step.py

```diff
@@ -51,7 +51,7 @@ class TweakRegStep(Step):
 
         # Object matching parameters:
         minobj = integer(default=15) # Minimum number of objects acceptable for matching
-        searchrad = float(default=1.0) # The search radius in arcsec for a match
+        searchrad = float(default=10.0) # The search radius in arcsec for a match
```

Comment:
I know we needed a value of 10" in order to get tweakreg to work on the simulated data that had bogus RA/Dec values, but I'm not sure we want to make that the default. Could such a large search radius ever cause harm when running it on data that have semi-accurate pointing info to begin with? I would think the default should be somewhere on the order of what we think the pointing uncertainty will be (with some buffer) and then let folks use non-default (larger) values when necessary.

Reply:
Fortunately, a large default value has no real bearing on run-time or accuracy of the final result unless, by some rare theoretical possibility, an extremely crowded field confuses the algorithm with a many spurious matches at various deltas (an occurance I have never seen or heard of, BTW).  Besides, for HST, I have seen delta's larger than 10 arcseconds (some have been determined to be over 60).  Granted, JWST should be more accurate, but this default should allow the pipeline to run on everything with, IMO, minimal/no risk of problems.  It can be reduced to original value if requested.

Reply:
👍